.github/actions/push-image/action.yml

```diff
@@ -51,7 +51,9 @@ runs:
       run: |
         curl -L https://github.com/lacework/lacework-vulnerability-scanner/releases/latest/download/lw-scanner-linux-amd64 -o lw-scanner
         chmod +x lw-scanner
-        ./lw-scanner image evaluate ${{ inputs.image_name }} ${{ inputs.image_tag }} --build-id ${{ github.run_id }} -v=false > report.json
+        ./lw-scanner image evaluate ${{ inputs.image_name }} ${{ inputs.image_tag }} --build-id ${{ github.run_id }} -v=false --fixable --disable-library-package-scanning > report.json
```

Comment:
What exactly do the `--fixable --disable-library-package-scanning` options do?

Reply:
Ah sorry, just saw it's related to Laceworks data collector. If we disable here, will it still be able to find issues with the image later on?

Reply:
Yeah the production images should be continually scanned by lacework in the background. 

We should also be able to replace this with explicit exceptions once we have access to their beta exceptions feature.